client/question/question.js

```diff
@@ -20,6 +20,71 @@ window.tie = angular.module('tie',
   ['ui.codemirror', 'tieConfig', 'tieData', 'ngCookies']);
 
 /**
+ * String used to label test suites that contain sample input and output
+ * that are presented to the user.
+ *
+ * @type {string}
+ */
+tie.constant('SAMPLE_INPUT', 'SAMPLE_INPUT'); 
```

Comment:
For the name of the constant, include what type of thing this is (in this case, it's a test suite ID, so maybe you could call it e.g. TEST_SUITE_ID_SAMPLE_INPUT).

Reply:
done